templates/security/index.html

```diff
@@ -477,7 +484,7 @@ <h2>Ubuntu Advantage</h2>
         <li class="p-list__item is-ticked">Knowledge Base: a private archive of expert-written articles and tutorials</li>
         <li class="p-list__item is-ticked">Support: phone and web-based support at multiple service levels</li>
       </ul>
-      <p><a href="/advantage" class="p-button--neutral">Purchase Ubuntu Advantage</a></p>
+      <p><a href="/advantage" class="p-button--positive">Purchase Ubuntu Advantage</a></p>
```

Comment:
```suggestion
      <p><a href="/advantage" class="p-button--positive">Visit the Ubuntu Advantage store</a></p>
```
Doesn't match copydoc

Reply:
I updated the copy doc in this case... someone forgot.